lib/modules/manager/types.ts

```diff
@@ -27,6 +27,10 @@ export interface RegexManagerTemplates {
   datasourceTemplate?: string;
   versioningTemplate?: string;
   depTypeTemplate?: string;
+  currentValueTemplate?: string;
```

Comment:
`RegexManagerTemplates` is partial duplicating `CustomManager` so we should unify them in a followup pr